### ISSUE TITLE
1.53.0 built from source has very few audio options

### ISSUE THREAD

**Issue by HWXLR8** [0ldsk00l/nestopia#432]
Version 1.53.0 built from 76c5d0cdb75444c54258a184eb7a488b8f1dd4ec on Gentoo Linux has very few options for audio in the "Settings" menu:

![image](https://github.com/user-attachments/assets/1cdfa77e-7225-4a54-bf9e-edd5787c71d3)

Am I missing something here? I would just like to mute the game audio.

**Comment by rdanbrook** [comment]
It would be a useful feature to add. I'll try to come up with the best way to implement it in the next few days. Until then, using this diff will have the intended effect:
```
diff --git a/source/fltkui/audiomanager.cpp b/source/fltkui/audiomanager.cpp
index 3713fec..5a4cc8c 100644
--- a/source/fltkui/audiomanager.cpp
+++ b/source/fltkui/audiomanager.cpp
@@ -172,6 +172,7 @@ int16_t AudioManager::dequeue() {
 }

 void AudioManager::queue(size_t in_size) {
+    return;
     spf = audinfo->spf;
     while ((bufsamples + spf) >= BUFSIZE) {
         SDL_Delay(1);
```
I also noticed you seem to have large text in the window title, but small text in the window itself. You can set the `FLTK_SCALING_FACTOR ` environment variable to 2 in order to use High DPI if you built against FLTK 1.4 and FLTK did not pick up the scaling factor from your window manager. You can also press Ctrl/+/-/0/.

**Comment by HWXLR8** [comment]
Thanks for the patch. I wouldn't mind helping out with the implementation if you don't have time.

Appreciate the tip about scaling as well.

**Comment by rdanbrook** [comment]
Mostly I need to think about how I want the setting to be controlled. I'm considering whether it should be like the fullscreen setting, where it can be enabled at startup and toggled during runtime. Also thinking about whether it should have a menubar item (which would need to work like the pause menu item, where text changes depending on state) or just have controls in the settings dialog. It will definitely make sense to add a hotkey for it too. Maybe it also needs a command line switch. Maybe everything I mentioned :).

**Comment by HWXLR8** [comment]
I would agree with "all of the above". I'll try to whip something up today, happy new year! :)
